target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+board=$(board_name)
+boardname="${board##*,}"
+
+case "$board" in
+mikrotik,routerboard-lhg-2nd)
+	ucidef_set_led_wlan "wlan" "wlan" "mikrotik:green:wlan" "phy0tpt"
```

Comment:
DT trigger?

Reply:
You mean it's not needed here as it's already in the DTS (`linux,default-trigger = "phy0tpt";`), right?

Reply:
Ah, indeed, didn't check whether you already had it in the DTS anyway. Yes, the DTS will do the job, so the line in 01_leds is just not necessary.

Reply:
Removed it from 01_leds. Thanks.